### PR TITLE
Resize image files in design introduction documents

### PR DESCRIPTION
**Pull request by s-k-song**
Image sizes are aligned.

Video links are updated.

### CODE REVIEW

File: docs/design/introduction/styles/iconography.md [hunk 1]
```diff
@@ -6,19 +6,19 @@ Tizen uses simple icon designs with subtle shadows. The following are guidelines
```
Comment (journey2w) [inline docs/design/introduction/styles/iconography.md]: There is a special character here. 0xC2. 

There is a special character where 3 blank lines are in this file. 
Please remove all special characters.

File: docs/design/introduction/styles/iconography.md [hunk 2]
```diff
@@ -6,19 +6,19 @@ Tizen uses simple icon designs with subtle shadows. The following are guidelines
 
  
 
-##### Use Icon Designs That Are Easy to Understand
+#### Use Icon Designs That Are Easy to Understand
 
 When you design icons, ensure that they are easily understood without any additional description. Where applicable, use universal designs that are familiar to users. It’s a good idea to perform a usability test on the icons after designing them.
```
Comment (journey2w) [inline docs/design/introduction/styles/iconography.md]: There is a special character here. 0xC2. Please remove it.

**Comment by s-k-song** [comment]
A new commit is applied to remove the special character 0xC2 by deleting 3 blank lines.
Also, some image tags are updated for proper alignment and display.

**Comment by s-k-song** [comment]
Video links are changed with \<video\> tags.
This commit is only for Design-introduction documents.
Other video links, say, in wearable documents, will be updated in other branches.

**Pull request merged**